src/core/connection.c

```diff
@@ -1522,6 +1534,7 @@ QuicConnTryClose(
         if (ResultQuicStatus) {
             Connection->CloseStatus = (QUIC_STATUS)ErrorCode;
             Connection->CloseErrorCode = QUIC_ERROR_INTERNAL_ERROR;
+            QuicPerfCounterIncrement(QUIC_PERF_COUNTER_CONN_CLOSED_ERROR);
```

Comment:
Let's call this event _SHUTDOWN_ERROR

Reply:
Though, I'm not convinced of the usefulness of this event, because connection idle is an "error" but will be the normal way HTTP connections are shutdown.

Reply:
Should it be scoped to non-idle errors? Or only app-level errors?

Reply:
I don't know... Maybe we call this _CONN_PROTOCOL_ERRORS instead and scope this to a subset of error codes?